src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -153,7 +153,7 @@ public FormulaValue NoElementValue(IRContext context)
 
             public FormulaValue GetDefault(IRContext context)
             {
-                return CommonErrors.DivByZeroError(context);
+                return CommonErrors.DivByZeroError(context, null);

```

Comment:
>null [](http://example.com/codeflow?start=60&length=4)

Do we need to pass the locale here (and in many other places of this file where there is a new `, null` argument added to the call?